docs/measurement/sci.md

```diff
@@ -1,6 +1,6 @@
 # Software Carbon Intensity
 
-SCI = ([E](https://sci-guide.greensoftware.foundation/E) * [I](https://sci-guide.greensoftware.foundation/I)) + [M](https://sci-guide.greensoftware.foundation/M) per [R](https://sci-guide.greensoftware.foundation/R)
+SCI = ([E](https://sci-guide.greensoftware.foundation/E) * [I](https://sci-guide.greensoftware.foundation/I) * [PUE](https://learn.greensoftware.foundation/energy-efficiency/#power-usage-effectiveness)) + [M](https://sci-guide.greensoftware.foundation/M) per [R](https://sci-guide.greensoftware.foundation/R)
```

Comment:
I would revert this as well to keep the top-level general SCI formula here. Then we can create a separate formula that showcases how we adapted the formula to the cloud-native scenario in our architecture. Something like this:

> Cloud-native SCI = ((CPU energy + memory energy) * PUE * static carbon coefficient) + embodied carbon of Equinix infrastructure 

Something like this 🤔 What do you think?

Reply:
To be clear, here we'll replace this into `SCI = (E + M) per R`? and then, in a separate section, develop the `E` to showcase the `Cloud-native SCI`?
Would we then set this "whole" formula in the dashboard as well or only in the docs?
As we're using the `container_total` from Kepler, which includes more than CPU and memory, I'd label it as "container energy".